maze4d/maze4d.vcxproj

```diff
@@ -185,29 +188,37 @@ xcopy /y /d  "$(ProjectDir)*.hlsl" "$(OutDir)"</Command>
     <ClCompile Include="MainUiController.cpp" />
     <ClCompile Include="Maze.cpp" />
     <ClCompile Include="MainMenu.cpp" />
+    <ClCompile Include="MazeField.cpp" />
```

Comment:
To satisfy CI build you also need to update a list of files in `CMakeLists.txt` and add all new `.cpp` files there.

Reply:
Thanks Andy, added them and resolved some conflicts.

Have never worked with CMake before, so I used CI action as debugger, which lead to some spamming commits-rebuilds, but seems it's fine now